print/layouts/no-vuetify.vue

```diff
@@ -1,120 +0,0 @@
-<template>
```

Comment:
Is deleting this also part of the pagedjs removal? Is there no way we will print without vuetify in the future? `v-calendar` for vuetify 3 is roughly at position 600 of 1004 in the column "New issues" (a column even more left than "Icebox"). I doubt that will be developed anytime soon...
https://app.zenhub.com/workspaces/vuetify-58dc72c478c79d245dc238f2/board
https://app.zenhub.com/workspaces/vuetify-58dc72c478c79d245dc238f2/issues/vuetifyjs/vuetify/13469

Reply:
I think removal of pagedjs and printing without vuetify is 2 independent topics. Latter would need, as you said, a proper replacement of v-calendar.